frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwarePackageCard/SoftwarePackageCard.tsx

```diff
@@ -108,6 +108,16 @@ const STATUS_DISPLAY_OPTIONS: Record<
     iconName: "pending-outline",
     tooltip: "Fleet is installing or will install when the host comes online.",
   },
+  pending_install: {
```

Comment:
@getvictor thank you for doing some frontend groundwork here. Looks like we want to merge the `pending_install` and `pending_uninstall` counts into a single indicator here, same with the `failed` statuses.

Reply:
I'll merge the numbers on the backend. Looks like we need two sets of statuses -- one for SoftwarePackageCard and one for Host details -> Software.